### ISSUE TITLE
Test with NVDA

### ISSUE THREAD

**Issue by TheJaredWilcurt** [TheJaredWilcurt/Koa11y#5]
NVDA is a free/open source screen reader that can be used for testing.

* https://www.nvaccess.org

Log any issues found to GitHub Issues page to be resolved.

**Comment by robgaston1** [comment]
Tested Koa11y using the Voiceover screen reader on Mac. The Voiceover screen reader does not work because of the version of NWJS that we are on. Voiceover will work with current version of NWJS, but devtools will not show up.
